website/docs/inheritdependencies.md

```diff
@@ -0,0 +1,27 @@
+inheritdependencies
+
+```lua
+inheritdependencies "value"
+```
+
+### Parameters ###
+
+`value` one of:
+* `On` - The project(s) will inherit library dependencies based on the parent project and project default settings. This is the default behavior.
+* `Off` - The project(s) will not inherit any library dependencies. Only explicitly specified dependencies will be linked.
+
+## Applies To ###
+
+The `config` scope.
+
+### Availability ###
+
+Visual Studio 2019 and later.
```

Comment:
Are you sure about this one?

Reply:
What makes you unsure?

Reply:
(Yes I'm sure)

Reply:
Also I added a mention of the `%(AdditionalDependencies)` tag.

Reply:
Is this really not available on earlier versions of Visual Studio?

Reply:
Oh that's what you meant. It probably is, but I can't test it well. It seems to exist since at earliest 2010.

Reply:
I'd be willing to change the documentation to say "at least Visual Studio 2015" if that's acceptable.

Reply:
It will be closer to the truth, so change it please. Thank you! 🤗

Reply:
Done.